src/App.tsx

```diff
@@ -1,9 +1,35 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const handleChangeQuery = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { value } = event.target;
+
+    setQuery(value);
+  };
+
+  const getLowerString = (str: string): string => {
+    return str.toLowerCase();
+  };
+
+  const handleSearchFilms = (films: Movie[]) => {
+    const lowerQuery = getLowerString(query).trim();
+
+    return films.filter(({ title, description }: Movie) => {
+      const lowerTitle = getLowerString(title);
+      const lowerDescription = getLowerString(description);
+
+      return lowerTitle.includes(lowerQuery)
+        || lowerDescription.includes(lowerQuery);
```

Comment:
you can move more logic in the helper function, so code will looks like :

`return helperFunction(title) || helperFunction(description)`